### PR TITLE
Convert the sidebar to ES6 syntax

### PR DESCRIPTION
**Pull request by timvandermeij**
*Easier reviewing with https://github.com/mozilla/pdf.js/pull/8306/files?w=1.*

### CODE REVIEW

File: web/pdf_sidebar.js [hunk 1]
```diff
@@ -53,23 +53,17 @@ var SidebarView = {
...
     this.isOpen = false;
     this.active = SidebarView.THUMBS;
     this.isInitialViewSet = false;
 
-    /**
-     * Callback used when the sidebar has been opened/closed, to ensure that
-     * the viewers (PDFViewer/PDFThumbnailViewer) are updated correctly.
-     */
+    // Callback used when the sidebar has been opened/closed, to ensure that
+    // the viewers (PDFViewer/PDFThumbnailViewer) are updated correctly.
```
Comment (Snuffleupagus) [inline web/pdf_sidebar.js]: Nit: I followed the same pattern as in [`PDFDocumentLoadingTask`](https://github.com/mozilla/pdf.js/blob/ece24dfe61d0cd6fe7a8b23728336efd4033fda7/src/display/api.js#L314-L351) when adding this comment, so unless you object violently my preference would be that we keep the formatting intact here :-)

**Review by Snuffleupagus — APPROVED** [review]
I've added one small nit, but otherwise this looks/works just fine; thank you for the patch!

File: web/pdf_sidebar.js (hunk 1 above)
Comment (timvandermeij) [inline web/pdf_sidebar.js]: Done in the new commit.

**Pull request merged**